zkevm-circuits/src/mpt_circuit.rs

```diff
@@ -934,11 +1005,14 @@ mod tests {
         dev::MockProver,
         halo2curves::{bn256::Fr, FieldExt},
     };
-
-    use std::fs;
+    use std::{fs, env::VarError};
 
     #[test]
     fn test_mpt() {
+        let only_run = var("ONLY_RUN")
+            .and_then(|idx| idx.parse::<usize>().map_err(|e|VarError::NotPresent)
+        ).ok();
+        println!("{:?}", only_run);
```

Comment:
Honestly for some reason I hadn't though of doing this myself, but this option makes a lot of sense!

Reply:
Missing `is_long` is because I thought we can represent the status of rlp values with two bytes:
```
 is_short   is_very_long
     true          false        short
     false         false        long
     false.        true        very_long
```